client/src/i18n/fr.json

```diff
@@ -1243,7 +1243,7 @@
     "ACCOUNT"       : "Rapport du Compte",
     "BALANCE"       : "Rapport de la Balance",
     "AGED_DEBTORS"  : {
-      "TITLE": "Aged Debtors",
+      "TITLE": "Dettes des clients",
```

Comment:
@patsonjunior had a better name for this.  "Detttes Agee des Clients" or something like that.  You should check with him on the name.